conscious/chain.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cvn",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://cvn.io/",
+  "pretty_name": "Cvn",
+  "chain_id": "canto_2032-1",
+  "bech32_prefix": "cvn",
+  "node_home": "$HOME/.cvnd",
+  "daemon_name": "cvnd",
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "acvn",
+        "fixed_min_gas_price": 100000000,
+        "low_gas_price": 100000000,
+        "average_gas_price": 200000000,
+        "high_gas_price": 300000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "acvn"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/cvn-network/cvn",
+    "recommended_version": "v1.0.0",
```

Comment:
this version does not exist in the Versions array. Did we mean to put v2.0.0 here?